nakone-law/src/components/Form/Form.jsx

```diff
@@ -0,0 +1,91 @@
+import './Form.css';
+import { Formik, Form, Field, ErrorMessage } from "formik";
+
+const initialValues = {
+  name: "",
+  phoneNumber: "",
+  email: "",
+  caseDescription: "",
+};
+
+const validate = (values) => {
+  const errors = {};
+
+  if (!values.name) {
+    errors.name = "Required";
+  }
+  if (!values.phoneNumber) {
+    errors.phoneNumber = "Required";
+  } else if (!/^\+?3?8?(0\d{9})$/i.test(values.phoneNumber)) {
+    errors.phoneNumber = "Invalid phone number";
+  }
+  if (!values.email) {
+    errors.email = "Required";
+  } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)) {
+    errors.email = "Invalid email address";
+  }
+  if (!values.caseDescription) {
```

Comment:
use YUP for Validation

Reply:
`const validationSchema = Yup.object().shape({
  name: Yup.string().required('Required'),
  phoneNumber: Yup.string().matches(/^\+?3?8?(0\d{9})$/, 'Invalid phone number').required('Required'),
  email: Yup.string().email('Invalid email address').required('Required'),
  caseDescription: Yup.string().required('Required'),
});`

Reply:
Thank you!